draft-ietf-quic-recovery.md

```diff
@@ -476,9 +476,10 @@ When ack-eliciting packets are in-flight in multiple packet number spaces,
 the timer MUST be set for the packet number space with the earliest timeout,
 except for ApplicationData, which MUST be ignored until the handshake
 completes; see Section 4.1.1 of {{QUIC-TLS}}.  Not arming the PTO for
-ApplicationData prioritizes completing the handshake and prevents the server
-from sending a 1-RTT packet on a PTO before before it has the keys to process
-a 1-RTT packet.
+ApplicationData prioritizes completing the handshake and prevents the client
```

Comment:
Just to clarify a bit more and clean up the run-on sentence.

ApplicationData prioritizes completing the handshake, prevents the client
from sending a 0-RTT packet on a PTO before it knows the server has accepted
0-RTT, and prevents the server from sending a 1-RTT packet on a PTO before
before it has the keys to process a 1-RTT acknowledgment.

Reply:
Accepted